src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -49,8 +47,32 @@
 
     public static final String PDP_XML_NAMESPACE = "http://www.kuali.org/kfs/pdp/payment";
     
-    public class ConcurPdpConstants {
-        public static final String CREDIT = "CR";
-        public static final String DEBIT = "DR";
+    public static class StandardAccountingExtractPdpConstants {
+        public static final String COMBINED_GROUP_INDICATOR = "Y";
+        public static final String BANK_CODE = "DISB";
+        public static final String EMPLOYEE_STATUS_CODE = "EMPLOYEE";
+        public static final String NON_EMPLOYEE_STATUS_CODE = "NON-EMPLOYEE";
+        public static final String EMPLOYEE_PAYEE_STATUS_TYPE_CODE = "E";
+        public static final String NON_EMPLOYEE_PAYEE_STATUS_TYPE_CODE = "Y"; 
+        public static final String FEED_FILE_ENTRY_HEADER_VERSION = "1.0";
+    }
+    
+    public static class ParameterNames {
```

Comment:
I think Chad had a comment about this too, let's move the parameters to a separate ConcurParameterConstants class